lib/controller/StudyController.js

```diff
@@ -26,6 +26,9 @@ StudyController.push({
   method: 'GET',
   path: '/studies/byCentreline/pending',
   options: {
+    auth: {
+      scope: [AuthScope.STUDY_REQUESTS.name],
```

Comment:
answer: no; Hapi's route schemas are validated by Joi, which doesn't accept `AuthScope` instances instead of strings here.

however, it *does* correctly match the `AuthScope` objects stored in `user.scope` against these, so that's good at least.